front/src/common/components/screen-reader-snackbar/screen-reader-snackbar.context.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { ScreenReaderSnackbarOptions } from './screeen-reader-snackbar.vm';
+
+interface Context {
+  options: ScreenReaderSnackbarOptions;
+  setOptions: (options: ScreenReaderSnackbarOptions) => void;
+}
+
+export const ScreenReaderSnackbarContext = React.createContext<Context>({
+  options: {
+    messages: [],
+  },
+  setOptions: options => {},
+});
+
+export const ScreenReaderSnackbarProvider: React.FunctionComponent = props => {
```

Comment:
Maybe here, we can pass from `props` the timeout (a number with milliseconds) to delete the messages, and add this timeout to options